### PR TITLE
Wallet UI: Initial Bootstrap app

### PR DESCRIPTION
**Pull request by bhumitaylor14**
https://linear.app/pillarproject/issue/PRO-2130/wallet-ui-initial-bootstrap

## Description

- Initial bootstrapping of react app through CRA
- Added config files.

## How Has This Been Tested?

- Tested on local

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by bhumitaylor14** [comment]
FYI: @IAmKio , added react app through CRA and added necessary config files, it is just basic app config, not included any dependancies which needed for etherspot UI component.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "wallet-ui",
```
Comment (IAmKio) [inline package.json]: Could we change this to `@etherspot/wallet-ui` please? This brings the package names in line with our other developer toolkits

File: package.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  "version": "0.1.0",
+  "description": "Wallet UI",
```
Comment (IAmKio) [inline package.json]: Please expand this description a little bit

File: public/manifest.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "Etherspot UI components",
+  "name": "Etherspot UI components Sample",
```
Comment (IAmKio) [inline public/manifest.json]:
```suggestion
  "short_name": "Wallet UI",
  "name": "A Wallet UI made up of Etherspot UI components",
```

File: package.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^16.18.76",
+    "@types/react": "^18.2.48",
+    "@types/react-dom": "^18.2.18",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
```
Comment (IAmKio) [inline package.json]: Can we lock these versions? This ensures that when other developers pull down the dependencies for this project, bugs don't creep in via patch releases for essential libraries

**Review by IAmKio — CHANGES_REQUESTED** [review]
Looking good, just some small changes 👍

**Comment by IAmKio** [comment]
@bhumitaylor14 please rename the title to make it accurate to what the PR is about

File: package.json (hunk 1 above)
Comment (bhumitaylor14) [reply package.json]: suggested changes done

File: package.json (hunk 2 above)
Comment (bhumitaylor14) [reply package.json]: Done

File: public/manifest.json (hunk 3 above)
Comment (bhumitaylor14) [reply public/manifest.json]: Changes done

File: package.json (hunk 4 above)
Comment (bhumitaylor14) [reply package.json]: Locked versions of all library

**Review by IAmKio — APPROVED** [review]

**Pull request merged**